src/MoBi.Assets/MoBi.Assets.csproj

```diff
@@ -26,9 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
-     <PackageReference Include="OSPSuite.Assets" Version="10.0.371-rsgvdwow" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="10.0.371-rsgvdwow" />
-    <PackageReference Include="OSPSuite.Core" Version="10.0.371-rsgvdwow" />
```

Comment:
@georgeDaskalakis @abdelr This is why we never reference those builds...I am not sure how we managed this but we did not release a version referencing released components...